pset-functions.js

```diff
@@ -136,6 +169,42 @@ console.log('Problem 3:')
 
 // Add your code below this line
 
+//Takes user input and calculates points
+function scrabbleScore(word) {
+   //Using split to convert user input to array of individual letters
+   const splitLetters = word.split('')
+   
+   //For loop that iterates through each array element and assigns a point value
+   for (i = 0; i < splitLetters.length + 1; i++) {
+      //If statement that assigns number to array element based on criteria
+      if (splitLetters[i] === 'a' || splitLetters[i] === 'e' || splitLetters[i] === 'i' || splitLetters[i] === 'o' || splitLetters[i] === 'u' || splitLetters[i] === 'l' || splitLetters[i] === 'n' || splitLetters[i] === 'r' || splitLetters[i] === 's' || splitLetters[i] === 't') {
+         splitLetters[i] = 1
+      } else if (splitLetters[i] === 'd' || splitLetters[i] === 'g') {
+         splitLetters[i] = 2
+      } else if (splitLetters[i] === 'b' || splitLetters[i] === 'c' || splitLetters[i] === 'm' || splitLetters[i] === 'p') {
+         splitLetters[i] = 3
+      } else if (splitLetters[i] === 'f' || splitLetters[i] === 'h' || splitLetters[i] === 'v' || splitLetters[i] === 'w' || splitLetters[i] === 'y') {
+         splitLetters[i] = 4
+      } else if (splitLetters[i] === 'k') {
+         splitLetters[i] = 5
+      } else if (splitLetters[i] === 'j' || splitLetters[i] === 'x') {
+         splitLetters[i] = 8
+      } else if (splitLetters[i] === 'q' || splitLetters[i] === 'z') {
+         splitLetters[i] = 10
+      }  else {
+         //Adds up the converted numbers to get total points
+         const add = (a, b) => a + b
+         const totalSum = splitLetters.reduce(add)
+         //console.log(totalSum)
+         //Returns the total points
+         return totalSum
+      }
+   }
+}
+
+const finalScore = scrabbleScore("cabbage")
+console.log(`Expected Result: ${finalScore}`)
+
```

Comment:
@cswebdesign Good Work here with this solution. Good use of the conditionals to compare each letter of the work with the letters of the alphabet and store the value in an array. Good job using the `reduce` method to add the points for each letter and return that value

Take a look at an alternate solution [here](https://github.com/jsr20190826/hw-02-functions-solution/blob/a4d789161886ff115657133c279062df8a4ca72d/pset-functions.js#L174)